### PR TITLE
Updating the 'Thank You' page

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Updated the 'Thank You' page to match the prototype. See [sketch](https://sketch.cloud/s/jP21z/a/G0npyb). Note: The feedback link is not required.

### How to review 
Check that the new Thank You page matches the prototype.

### CODE REVIEW

**Comment by berroar** [comment]
[Post submission page (S)](https://trello.com/c/ad8MFrRw/3019-post-submission-page-s)

**Review by bitdivision — COMMENTED** [review]
Looks good :+1:

You will need to also update the static translations to include the new strings.

**Comment by bitdivision** [comment]
I would also be surprised if a few functional tests didn't fail now.

**Comment by berroar** [comment]
> I would also be surprised if a few functional tests didn't fail now.

Have now removed the only functional test that was failing.

File: tests/integration/routes/test_view_submission.py [hunk 1]
```diff
@@ -1,363 +0,0 @@
-import simplejson as json
```
Comment (bitdivision) [inline tests/integration/routes/test_view_submission.py]: If we're removing the test for this, then we should probably remove the code that handles view submission as well.

Comment (bitdivision) [reply tests/integration/routes/test_view_submission.py]: Just spoke to @ajmaddaford about this. Suggestion is to leave the view submission feature in runner. So in that case, we should leave the tests here as well.

You will probably need to just remove the references to the thank you page and go directly to the view submission URL.

Comment (berroar) [reply tests/integration/routes/test_view_submission.py]: Have put this file back and removed references to the thank you page.

**Review by JamesGardiner — APPROVED** [review]

**Review by iwootten — APPROVED** [review]

File: tests/integration/routes/test_view_submission.py [hunk 2]
```diff
@@ -29,10 +29,6 @@ def setUp(self):
 
     def test_view_submission(self):
```
Comment (iwootten) [inline tests/integration/routes/test_view_submission.py]: I've just noticed that this test is no longer asserting anything, which suggests it should just be completely removed.

File: tests/integration/routes/test_view_submission.py [hunk 3]
```diff
@@ -88,16 +84,9 @@ def test_try_view_submission_when_not_available(self):
         # Submit answers
         self.post(action=None)
 
-        # check we're on the thank you page and view submission link is not available
```
Comment (iwootten) [inline tests/integration/routes/test_view_submission.py:91]: This test method name no longer matches what is actually being tested. It should end on an assertion. I think the only thing that needed to be removed was the "self.assertNotInBody"?

**Pull request merged**